back/app/controllers/web_api/v1/events_controller.rb

```diff
@@ -118,6 +118,39 @@ def event_params
     end
   end
 
+  def finder_params
+    params.tap do |p|
+      if p.key?(:ongoing_during)
+        p[:ongoing_during] = parse_date_range(p[:ongoing_during])
+      end
+    end
+  end
+
+  def parse_date_range(date_range)
+    raise ArgumentError, 'date_range must be a String' unless date_range.is_a?(String)
+
+    date_range = date_range.reverse.chomp('[').reverse.chomp(']')
+    # Ignore extra items if there are more than two. ("Be conservative in what you send,
+    # and liberal in what you accept." — at least for now)
```

Comment:
My Uni training tells me this should be cited... https://en.wikipedia.org/wiki/Robustness_principle or 'Postel's law', or just add `, Jon Postel`.